test_runner/performance/test_lazy_startup.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import requests
+from fixtures.benchmark_fixture import MetricReport, NeonBenchmarker
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+# Just start and measure duration.
```

Comment:
please, update this comment to reflect the difference with `test_startup_simple`

Reply:
Done